app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -0,0 +1,64 @@
+module Idv
+  class HowToVerifyController < ApplicationController
+    include IdvStepConcern
+    include RenderConditionConcern
+
+    before_action :confirm_step_allowed
+
+    check_or_render_not_found -> { self.class.enabled? }
```

Comment:
I am not seeing a not found page when I type in the url. We displayed the 404 view when we worked on the non-FSM views. I think it is a good idea. I am not sure this is working.